### ISSUE TITLE
Feature Request: Display files in `stash` directory

### ISSUE THREAD

**Issue by this-is-yaash** [AmateursLeague/sneaky-package#99]
Add a method `ls()` inside `show.py` that displays the files list inside the `stash` directory when executed.

**Example:**

```python

from package.show import display
display()
```
**Output**:
```
1. file1
2. file2
3. file3
4. file4
5. file5
6. file6
7. file7
8. file8
```

**Explanation**:
- As you can see the above code, it doesn't explicitly call the `ls()` function.
- The feature should execute `ls()` function when there isn't any arguments passed.

_In simple terms_, 
- Implement `ls()` method, that list files inside `stash` directory in python execution console.
- It should execute when `display()` is called without any arguments.

**Comment by this-is-yaash** [comment]
Make sure the assignees, use the development branch(when assigned).

Execute the following command, after forking the repo and initializing local environment.
```bash
git fetch origin
git checkout 99-feature-request-display-files-in-stash-directory
```

**Comment by Prakhar29Sharma** [comment]
I would like to work on this issue

**Comment by this-is-yaash** [comment]
@Prakhar29Sharma please explain on how you are planning to solve this issue?

**Comment by Prakhar29Sharma** [comment]
> @Prakhar29Sharma please explain on how you are planning to solve this issue?

Will add the ls() method in the models.py. use os package to get to stash dir and list files. print files name in the provided format

**Comment by Wahid7852** [comment]
Assigning the issue to you @Prakhar29Sharma. Good Luck!

**Comment by Wahid7852** [comment]
You've a time duration of 24 hours to complete this issue since it is a mere level 1 issue.

**Comment by Prakhar29Sharma** [comment]
Hey I have a doubt, I've made changes to the models.py file in build/lib/package directory but since build/ directory is added to .gitignore file I'm not able to see the changes I've made through git status.

**Comment by Prakhar29Sharma** [comment]
Hey @Wahid7852 , I was facing some issue with merge conflict so I've closed the previous PR and created a new one with the requested changes, please review it

**Comment by debabrata-15** [comment]
Hey @this-is-yaash I would like to work on this issue
Here's my idea 
adding a ls() function in show.py and then calling it from display.
Thank You

**Comment by this-is-yaash** [comment]
Open to all! Please comment explaining your approach to get assigned!

**Comment by yashksaini-coder** [comment]
Can i work on this issue @this-is-yaash @jindalpriyanshu101

**Comment by jindalpriyanshu101** [comment]
> Can i work on this issue @this-is-yaash @jindalpriyanshu101

Sure, just explain your approach. if its satisfactory, you'll be assigned.

**Comment by debabrata-15** [comment]
> Open to all! Please comment explaining your approach to get assigned!

Okay Here's my Idea of doing it 
creating a ls() method passing directory location as it's argument show.py and calling the function in display.py

**Comment by this-is-yaash** [comment]
> > Open to all! Please comment explaining your approach to get assigned!
> 
> Okay Here's my Idea of doing it creating a ls() method passing directory location as it's argument show.py and calling the function in display.py

Could you please explain this in more detail, using logical terms on how you are planning to implement the method? @debabrata-15

**Comment by debabrata-15** [comment]
> > > Open to all! Please comment explaining your approach to get assigned!
> > 
> > 
> > Okay Here's my Idea of doing it creating a ls() method passing directory location as it's argument show.py and calling the function in display.py
> 
> Could you please explain this in more detail, using logical terms on how you are planning to implement the method? @debabrata-15
Sure, I will define a methods ls() and display() in show.py . ls() will take the stash file directory as an argument and  display() will call ls() when no arguments is passed.

**Comment by this-is-yaash** [comment]
@debabrata-15 your approach seems quite convincing.
It's been 4 days... would you still like to work on this issue?

**Comment by debabrata-15** [comment]
> @debabrata-15 your approach seems quite convincing.
> It's been 4 days... would you still like to work on this issue?

Yes I would like to work on this

**Comment by this-is-yaash** [comment]
@debabrata-15 you are assigned! 
You have to submit the PR before tomorrow EOD.

**Comment by Wahid7852** [comment]
@debabrata-15 Updates?

**Comment by debabrata-15** [comment]
> @debabrata-15 Updates?

I did submit my PR but there's some issue. A lot of commits are getting pushed it seems which are not mine

**Comment by this-is-yaash** [comment]
@debabrata-15 
Your PR overlaps with the previous commits.

1. Sync your forked repoitory of `sneaky-package` with upstream branch.
2. Switch to main branch `git switch to main`.
3. Pull the lastest changes (just in case) `git pull origin main`
4. Now you have the latest changes updated in your local `main` branch. 
5. Create new on top of `main` branch. `git switch main` & `git switch -c yourNewBranchName`.
6. At last, you have a new branch to work on the issue that you were assigned on!


> I think you are unfamiliar with git. Try deleting the `sneaky-package` in your local and clone again. with the updated steps.

**Comment by this-is-yaash** [comment]
@debabrata-15 although you can fix it easily by `rebasing`, if you know how to use it. 
But try this manual way.

**Comment by this-is-yaash** [comment]
fixes #159
